src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx

```diff
@@ -212,7 +216,6 @@ export default function EarnPoolInfoScreen({ route, navigation }: Props) {
   }, [allPositionsWithBalance])
 
   const onPressWithdraw = () => {
-    // TODO(tomm): once act-1385 is merge use the bottom sheet button presses
```

Comment:
This is covered by the event [`EarnEvents.earn_select_withdraw_type`](https://github.com/valora-inc/wallet/blob/main/src/analytics/docs.ts#L613).